src/Components/Atoms/CheckBox/index.tsx

```diff
@@ -1,12 +1,12 @@
 import { useState } from 'react'
-import style from './CheckBoxPattern.module.scss'
+import style from './CheckBox.module.scss'
 
 type Props = {
     checked?: boolean
     onChange?: (isChecked: boolean) => void;
```

Comment:
Deixa esses dois como obrigatorios, props q são essenciais para o funcionamento do componente devem ser passadas sempre como obrigatorias
```suggestion
    checked: boolean
    onChange: (isChecked: boolean) => void;
```